PWGCF/EbyEFluctuations/Tasks/RobustFluctuationObservables.cxx

```diff
@@ -0,0 +1,1965 @@
+
+  // ##### configurables
+  Configurable<int> nBinsPt{"nBinsPt", 800, "N bins in pT histo"};
+  Configurable<int> nBinsEta{"nBinsEta", 400, "N bins in eta histo"};
+  Configurable<int> nBinsPhi{"nBinsPhi", 360, "N bins in phi histo"};
+  Configurable<float> vtxCut{"vtxCut", 10.0, "Accepted z-vertex range (cm)"};
+  Configurable<int> flagPbPb{"flagPbPb", 0, "0 - pp, 1 - PbPb"};
+
+  // orbit QA
+  uint32_t orbitAtCollIndexZero = 0;
+
+  // DF QA
+  map<uint64_t, uint64_t> mapDF;
+
+  OutputObj<TH1D> h2D_Orbit_vs_CollIndex_0{TH1D("h2D_Orbit_vs_CollIndex_DF0", "h2D_Orbit_vs_CollIndex_DF0;collision index;orbit", 10001, -0.5, 10000.5)};
+  OutputObj<TH1D> h2D_Orbit_vs_CollIndex_1{TH1D("h2D_Orbit_vs_CollIndex_DF1", "h2D_Orbit_vs_CollIndex_DF1;collision index;orbit", 10001, -0.5, 10000.5)};
+  OutputObj<TH1D> h2D_Orbit_vs_CollIndex_2{TH1D("h2D_Orbit_vs_CollIndex_DF2", "h2D_Orbit_vs_CollIndex_DF2;collision index;orbit", 10001, -0.5, 10000.5)};
+  OutputObj<TH1D> h2D_Orbit_vs_CollIndex_3{TH1D("h2D_Orbit_vs_CollIndex_DF3", "h2D_Orbit_vs_CollIndex_DF3;collision index;orbit", 10001, -0.5, 10000.5)};
+  OutputObj<TH1D> h2D_Orbit_vs_CollIndex_4{TH1D("h2D_Orbit_vs_CollIndex_DF4", "h2D_Orbit_vs_CollIndex_DF4;collision index;orbit", 10001, -0.5, 10000.5)};
+
+  OutputObj<TH1D> h2D_BC_vs_CollIndex_0{TH1D("h2D_BC_vs_CollIndex_DF0", "h2D_BC_vs_CollIndex_DF0;collision index;orbit", 10001, -0.5, 10000.5)};
+  OutputObj<TH1D> h2D_BC_vs_CollIndex_1{TH1D("h2D_BC_vs_CollIndex_DF1", "h2D_BC_vs_CollIndex_DF1;collision index;orbit", 10001, -0.5, 10000.5)};
+  OutputObj<TH1D> h2D_BC_vs_CollIndex_2{TH1D("h2D_BC_vs_CollIndex_DF2", "h2D_BC_vs_CollIndex_DF2;collision index;orbit", 10001, -0.5, 10000.5)};
+  OutputObj<TH1D> h2D_BC_vs_CollIndex_3{TH1D("h2D_BC_vs_CollIndex_DF3", "h2D_BC_vs_CollIndex_DF3;collision index;orbit", 10001, -0.5, 10000.5)};
+  OutputObj<TH1D> h2D_BC_vs_CollIndex_4{TH1D("h2D_BC_vs_CollIndex_DF4", "h2D_BC_vs_CollIndex_DF4;collision index;orbit", 10001, -0.5, 10000.5)};
+
+  void init(InitContext const&)
+  {
+    // cuts on phi vs pt (to avoid TPC boundaries)
+    fPhiCutExpPosHigh = new TF1("fPhiCutExpPosHigh", "[0]*exp([1]*x)+[2]", 0, 20);
+    fPhiCutExpPosLow = new TF1("fPhiCutExpPosLow", "[0]*exp([1]*x)+[2]", 0, 20);
+
+    fPhiCutExpNegHigh = new TF1("fPhiCutExpNegHigh", "[0]*exp([1]*x)+[2]", 0, 20);
+    fPhiCutExpNegLow = new TF1("fPhiCutExpNegLow", "[0]*exp([1]*x)+[2]", 0, 20);
+
+    fPhiCutExpPosHigh->SetParameters(0.3, -1, 0.055 + constPhiShift);
+    fPhiCutExpPosLow->SetParameters(0.15, -1, -0.02 + constPhiShift);
+
+    fPhiCutExpNegHigh->SetParameters(-0.15, -1, +0.02 + constPhiShift);
+    fPhiCutExpNegLow->SetParameters(-0.3, -1, -0.055 + constPhiShift);
+
+    // ### event-wise:
+    // AxisSpec axisNcontrib{flagPbPb ? 8001 : 501, -0.5, flagPbPb ? 8000.5 : 500.5, "n vertex contributors"};
+    // AxisSpec axisNtracks{flagPbPb ? 8001 : 501, -0.5, flagPbPb ? 8000.5 : 500.5, "n tracks"};
```

Comment:
Please, as an overall rule in O2, don't leave code commented
It is confusing if someone wants to learn on what you have done
I saw in some cases you comment a section of code apparently for preserving it either for your  knowledge or to latter use. In gitHub, you know, there is a right place for doing that: gitHub gists, https://gist.github.com/altsybee